content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -88,7 +88,7 @@ different Kubernetes components.
 | `DefaultPodTopologySpread` | `false` | Alpha | 1.19 | |
 | `DevicePlugins` | `false` | Alpha | 1.8 | 1.9 |
 | `DevicePlugins` | `true` | Beta | 1.10 | |
-| `DisableAcceleratorUsageMetrics` | `false` | Alpha | 1.19 | 1.20 |
```

Comment:
Leave this line in (to show the history of this feature gate).

Reply:
Done!